keycloak-rest-api-extensions-tests/src/test/java/io/cloudtrust/keycloak/credential/Ssha256PasswordHashProviderTest.java

```diff
@@ -0,0 +1,32 @@
+package io.cloudtrust.keycloak.credential;
```

Comment:
I am wondering if we should also make a keycloak integration test and not only the standard unit test.